use-cases/time-series-forecasting/walmart-sales/EDA/notebooks/EDA.ipynb

```diff
@@ -0,0 +1,384 @@
+   "source": [
+    "plt.figure()\n",
+    "sales_vs_dept = sns.lineplot(\n",
+    "    x=\"Dept\", y=\"Weekly_Sales\", hue=\"Year\", data=df, palette=[\"b\", \"g\", \"orange\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure()\n",
+    "sales_vs_storeType = sns.boxenplot(\n",
+    "    x=\"Type\", y=\"Weekly_Sales\", data=df, hue=\"IsHoliday\", palette=[\"b\", \"orange\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure()\n",
+    "sales_vs_temp = sns.histplot(data=df.groupby(\"Weekly_Sales\")[\"Temperature\"].mean(), bins=30)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_normalized = normalize_data(df, \"Weekly_Sales\")\n",
+    "plt.figure()\n",
+    "sales_vs_storeType_normalized = sns.boxenplot(\n",
+    "    x=\"Type\",\n",
+    "    y=\"Weekly_Sales\",\n",
+    "    data=df_normalized,\n",
+    "    hue=\"IsHoliday\",\n",
+    "    palette=[\"b\", \"orange\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Log dataset visualizations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run[\"data/visualizations/corr_matrix\"].upload(File.as_image(sns_corr_matrix.figure))\n",
+    "run[\"data/visualizations/sales_vs_week\"].upload(File.as_image(sales_vs_week.figure))\n",
+    "run[\"data/visualizations/sales_vs_month\"].upload(File.as_image(sales_vs_month.figure))\n",
+    "run[\"data/visualizations/sales_vs_quarter\"].upload(File.as_image(sales_vs_qt.figure))\n",
+    "run[\"data/visualizations/sales_vs_store\"].upload(File.as_image(sales_vs_store.figure))\n",
+    "run[\"data/visualizations/sales_vs_dept\"].upload(File.as_image(sales_vs_dept.figure))\n",
+    "run[\"data/visualizations/sales_vs_IsHoliday\"].upload(File.as_image(sales_vs_isholiday.figure))\n",
+    "run[\"data/visualizations/sales_vs_temperature\"].upload(File.as_image(sales_vs_temp.figure))\n",
+    "run[\"data/visualizations/sales_vs_storeType\"].upload(File.as_image(sales_vs_storeType.figure))\n",
+    "run[\"data/visualizations/sales_vs_storeType_normalized\"].upload(\n",
+    "    File.as_image(sales_vs_storeType_normalized.figure)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Stop logging\n",
+    "\n",
+    "Once you are done logging, stop tracking the run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run.stop()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
```

Comment:
```suggestion
 "metadata": {
  "keep_output": true,
  "kernelspec": {
```

needed to prevent nbstripout from stripping outputs

Reply:
??